app/models/provider/synth.rb

```diff
@@ -0,0 +1,51 @@
+class Provider::Synth
+  include Retryable
+
+  def initialize(api_key = ENV["SYNTH_API_KEY"])
+    @api_key = api_key || ENV["SYNTH_API_KEY"]
+  end
```

Comment:
This looks like we're doubly guarding for the same thing. We're not. But it's an admittedly slightly premature guard.

When we ultimately pass in `ENV["EXCHANGE_RATE_PROVIDER_API_KEY"]` — that value might be nil.

If we want to both be able to instantiate this without any args (`Provider::Synth.new`) and also be able to pass in a possibly `nil` key, I think this would be the way to support that.